ea_load_trajectory.m

```diff
@@ -1,4 +1,16 @@
 function pobj=ea_load_trajectory(directory,side)
+% 
```

Comment:
@ningfei Please provide a description of the function, the inputs and the outputs.